CHANGELOG.md

```diff
@@ -5,9 +5,8 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project does NOT adhere to [Semantic Versioning](https://semver.org/spec/v2.0.0.html) and packages shipped from the beta service library will always be in `preview` as changes to the API are for non production use and commonly ship breaking changes.
 
-## [Unreleased]
 
-## [5.78.0-preview] - 2024-06-06
+## [5.78.0] - 2024-06-06
```

Comment:
we need to keep the -preview. Please make sure release please is configured accordingly